modules/CCBR/fastqc/meta.yml

```
@@ -0,0 +1,57 @@
+name: fastqc
+description: Run FastQC on sequenced reads
```

For modules adapted from nf-core, I have been adding "Adapted from [permalink]" to the description

```suggestion
description: Run FastQC on sequenced reads. Adapted from https://github.com/nf-core/modules/tree/73a6d7e6077b88aba1c5d6805635d79d6718270c/modules/nf-core/fastqc
```